packages/grafana-toolkit/README.md

```diff
@@ -216,6 +216,9 @@ module.exports = {
 ### How do I add 3rd party dependencies that are not npm packages?
 You can add such dependencies by putting them in `static` directory in the root of your project. The `static` directory will be copied when building the plugin.
 
+### I have encountered `Unable to dynamically transpile ES module A loader plugin needs to be configured via SystemJS.config({ transpiler: 'transpiler-module' }).`. What should I do?
+This error can occur when you bundle your plugin using `grafana-toolkit plugin:dev` task. This task does not remove comments from the bundled package, so **make sure your comments does not include ES2016 code**. If they do, SystemJS v0.20.19 which Grafana uses internally to load plugins, will interprete your code as an [ES module](https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/). Grafana does not support ESM.
+
 ## Contribute to grafana-toolkit
 You can contribute to grafana-toolkit in the by helping develop it or by debugging it.
```

Comment:
```suggestion
You can contribute to grafana-toolkit by helping develop it or by debugging it.
```

Reply:
Please add a link to the appropriate document or documents in the `contribute` folder.